### PR TITLE
Fix: 1271 sig validation chainId validation

### PR DESCRIPTION
**Pull request by ganchoradkov**
## Description
Fixed a bug where we were not passing CAIP2 formatted `chainId` for 1271 signature validation 
## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?
tests

## Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Additional Information (Optional)

Please include any additional information that may be useful for the reviewer.

### CODE REVIEW

File: packages/utils/src/signatures.ts [hunk 1]
```diff
@@ -49,6 +50,12 @@ export async function isValidEip1271Signature(
   projectId: string,
   baseRpcUrl?: string,
 ) {
+  const parsedChain = parseChainId(chainId);
+  if (!parsedChain.namespace || !parsedChain.reference) {
+    throw new Error(
+      `isValidEip1271Signature failed: chainId must be in CAIP-2 format, received: ${chainId}`,
```
Comment (Cali93) [inline packages/utils/src/signatures.ts:56]: Why should it be in CAIP-2 format since EIP-1271 signature is specific to EVM ?

**Review by Cali93 — COMMENTED** [review]
Curious why this is needed in the first place ? Shouldn't we process the verification even if we don't have a namespace ?

File: packages/utils/src/signatures.ts (hunk 1 above)
Comment (ganchoradkov) [reply packages/utils/src/signatures.ts:56]: blockchain api rejects the request if it isnt. Probably we can update there it there.

Comment (Cali93) [reply packages/utils/src/signatures.ts:56]: Wouldn't this be a breaking change if someone provides a custom RPC url and is sending the non compliant CAIP-2 chain ID ?

**Review by Cali93 — APPROVED** [review]

File: packages/utils/src/signatures.ts (hunk 1 above)
Comment (ganchoradkov) [reply packages/utils/src/signatures.ts:56]: it shouldn't. These params are opinionated for our rpc api. Direct node urls do not use them

**Pull request merged**